app/models/category.rb

```diff
@@ -0,0 +1,7 @@
+class Category < ApplicationRecord
```

Comment:
- [ ] Kindly change this association to match the ERD guidelines, a category should belong to author like so `belongs_to :author, class_name: User`. This change should be reflected in the scheme. Categories table should have an `author_id` instead of `user_id`.

Reply:
**Hi @elisha2kyakpo1,**

Thanks for your review. But about the `author_id` since I used customized entity names that's why I used `user_id` and I reached out to my peers and asked how they used every one of them used `user_id` and get approved. Kindly, can you check my customized entities it's matching the ERD diagram.